packages/cloudflare/src/cli/templates/worker.ts

```diff
@@ -21,6 +21,21 @@ const cloudflareContextALS = new AsyncLocalStorage<CloudflareContext>();
   }
 );
 
+async function applyProjectEnvVars(mode = "production") {
```

Comment:
Would doing this once at top-level remove the need for `process.env.__OPENNEXT_PROCESSED_ENV` ?

nit: maybe rename secrets to nextEnvVars? (those are not all secrets, i.e. `NEXT_PUBLIC_`)

Reply:
We can't do this at the top-level because of the desire to have the `NEXTJS_ENV` .dev.vars variable. If that's not needed, then I can move it to the top-level